SPECS/librepo/librepo.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Repodata downloading library
 Name:           librepo
-Version:        1.14.2
-Release:        2%{?dist}
+Version:        1.15.1
```

Comment:
Given the bump here, there may be compat concerns.  Does DNF or TDNF (or any other package need to be dash-version-bumped to force a recompile against this?  

Is there any documentation for librepo that indicates compatibility issues moving from 1.14 to 1.15?

Reply:
it looks like there are a couple other packages that use librepo. libdnf, and rpm-ostree appear to be the only other packages using it. It may be worth dash-rolling these packages just to make sure nothing is missing from them.

Here is the release notes from the release of librepo-1.15.1: https://github.com/rpm-software-management/librepo/releases/tag/1.15.1
As far as I can tell, this is essentially only a new version due to additions to the API, rather than a particular breaking change.